An accessor cannot be declared in an ambient context.

I recently updated firebase-functions and all fire base related packages, but I got an error : `An accessor cannot be declared in an ambient context.` when I tried to build my project.

before this update my project run and deployed successfully to firebase, no any other modification made.

error details:
```
> tsc -p firebase/tsconfig.json 
```

```
node_modules/firebase-functions/lib/apps.d.ts(26,13): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/firebase-functions/lib/apps.d.ts(33,21): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/firebase-functions/lib/function-builder.d.ts(53,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/firebase-functions/lib/function-builder.d.ts(66,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/firebase-functions/lib/function-builder.d.ts(100,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/firebase-functions/lib/function-builder.d.ts(114,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/firebase-functions/lib/function-builder.d.ts(121,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/firebase-functions/lib/function-builder.d.ts(128,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/firebase-functions/lib/function-builder.d.ts(137,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/firebase-functions/lib/function-builder.d.ts(150,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/firebase-functions/lib/function-builder.d.ts(159,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/firebase-functions/lib/function-builder.d.ts(165,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/firebase-functions/lib/handler-builder.d.ts(15,9): error TS1086: An accessor cannot be declared in an ambient context.
```

![jpg](https://user-images.githubusercontent.com/57308531/80095722-af28c880-8568-11ea-93b2-8f1055089342.jpg)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,0 +1,1 @@
+- Update TypeScript dependency to fix build issues (Issue #667)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,1 +1,1 @@
-- Update TypeScript dependency to fix build issues (Issue #667)
+- Update TypeScript dependency to v.3.8 to fix build issues (Issue #667)

--- a/package.json
+++ b/package.json
@@ -74,7 +74,7 @@
     "tslint-no-unused-expression-chai": "^0.1.4",
     "tslint-plugin-prettier": "^2.0.1",
     "typedoc": "^0.17.1",
-    "typescript": "^3.5.2",
+    "typescript": "^3.8.3",
     "yargs": "^15.3.1"
   },
   "peerDependencies": {

